product_standard_margin/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Product Margin and Margin Rate",
-    "version": "16.0.1.0.1",
+    "version": "16.0.1.0.2",
```

Comment:
This is unncessary. Will be done automatically, once merged.

Reply:
noted @legalsylvain